src/app.py

```diff
@@ -22,7 +22,7 @@
 from utils.api_tools import surgeo_helper, zrp_helper
 
 app = Flask(__name__)
-CORS(app)
+CORS(app, resources={r"/surgeo/*": {"origins": "*"}, r"/zrp/*": {"origins": "*"}})
```

Comment:
Can you write up a ticket with all the details of the CORS bug? That's something we should address, if we're going to be adding additional endpoints.

Reply:
Also, no change necessary, but you explain real quick what this change to CORS() does here? You can just reply to the comment. 

I want to know to keep this in mind for when we add additional endpoints.

Reply:
I actually don't think the error exists anymore. There was some missing brace in the get request. But before, there was a "No 'Access-Control-Allow-Origin'" error, where the api call just wasn't working for the ZRP. This shouldn't be necessary and I can remove this now